### PR TITLE
Fixes #25029 - Allow revert template to original state

### PR DESCRIPTION
**Pull request by domitea**

### CODE REVIEW

**Comment by ares** [comment]
ok to test

**Comment by ares** [comment]
works fine :+1:

File: app/helpers/audits_helper.rb [hunk 1]
```diff
@@ -269,7 +269,12 @@ def rebuild_audit_changes(audit)
     css_class_name = css_class_by_action(audit.action == 'destroy')
     audit.audited_changes.map do |name, change|
       next if change.nil? || change.to_s.empty?
-      next if name == 'template'
+      # update data for created template for better view
+      if name == 'template' && audit.action == 'create'
+        audit.audited_changes['template'] = ['', audit.audited_changes['template']]
+      else
+        next
```
Comment (ezr-ondrej) [inline app/helpers/audits_helper.rb]: I believe there is a logical mistake.
The next was called for `name == 'template'`, now we are skipping everything apart of `name == 'template' && ...`. That's probably not what you've meant, right?
I believe we should still be skipping only if template, we should just add the changes if it's `create`. Considering we are touching the iterated array, wouldn't it be easier just to move the change before (or after) the loop and keep the loop itself untouched?

**Review by ezr-ondrej — CHANGES_REQUESTED** [review]
There is probably a mistake, otherwise it makes sense :+1:

File: app/helpers/audits_helper.rb (hunk 1 above)
Comment (domitea) [reply app/helpers/audits_helper.rb]: Sorry, my bad.

**Review by ezr-ondrej — APPROVED** [review]
Works as expected, nice addition!
Thanks @domitea! :+1:

**Pull request merged**